src/app/store/slices/storage/storage.thunks/uploadItemsThunk.ts

```diff
@@ -61,8 +61,10 @@ export const uploadItemsThunk = createAsyncThunk<void, UploadItemsPayload, { sta
     try {
       const planLimit = getState().plan.planLimit;
       const planUsage = getState().plan.planUsage;
+      const uploadItemsSize = Object.values(files).reduce((acum, file) => acum + file.size, 0);
+      const totalItemsSize = uploadItemsSize + planUsage;
 
-      if (planLimit && planUsage >= planLimit) {
+      if (totalItemsSize >= planLimit) {
```

Comment:
It seems that has removed the `planLimit` truly checking before the comparison with the usage. It is not necessary that checking?